### ISSUE TITLE
Host doesn't see mounted free space on external drive

### ISSUE THREAD

**Issue by JaspervdK** [TRON-US/go-btfs#986]
Hi, I was trying to set up a new host, using a mounted external drive on my Ubuntu server as the storage location. 
I was able to correctly set this up, but the host won't let me set a higher max storage amount than the free space on internal storage, which is only about 100GB.
Setting a higher maximum by running `btfs storage announce --host-storage-max=500` also doesn't work: `Error: new max storage size is invalid (exceeds available space)`
Is there a known way around this, or is this currently not possible?

**Comment by JaspervdK** [comment]
Refreshing an re-selecting the mounted drive didn't seem to work.
![image](https://user-images.githubusercontent.com/22342867/114146651-27be9b80-9918-11eb-852d-a82dde54e9a3.png)

I mounted the drive to a folder in a user folder (`/home/jasper/BTFSDriveMount`) to avoid problems with permissions and then made folder on the drive `/home/jasper/BTFSDriveMount/BTFS` which I selected as the location in the hostui. Maybe BTFS is not seeing the size of the external drive because of this setup?
